assets/css/style.css

```diff
@@ -339,3 +352,265 @@ fieldset > textarea:focus {
+    margin-right: 50px;
+    text-decoration: none;
+    color: #1a2236;
+  }
+
+  #menu-wrapper > a:hover {
+    text-decoration: underline #1a2236 2px;
+  }
+
+  .headline > .text-wrapper {
+    max-width: 55%;
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    background-image: url('/assets/img/image_dots.png');
+    background-repeat: no-repeat;
+    background-size: 20vh;
+    background-position: bottom right;
+  }
+
+  .headline > .text-wrapper > p {
+    margin-right: 35%;
+  }
+
+  .button {
+    width: 15em;
+    padding: 1.3em;
+  }
+
+  .headline {
+    height: 100vh;
+    justify-content: center;
+    padding: 0 10%;
+    background-image: url('/assets/img/image_geometry_3.png');
+    background-size: 45% 100vh;
+    background-position: right;
+  }
+
+  h1 {
+    font-size: 8em;
+    margin: 0;
+  }
+
+  h2 {
+    font-size: 8em;
+  }
+
+  .projects {
+    background-image: url('/assets/img/image_dots.png');
+    background-repeat: no-repeat;
+    background-position: 430px 210px;
+    background-size: 20vh;
+  }
+
+  .projects > h2 {
+    font-size: 7em;
+    transform: rotate(-90deg) translate(400px, -750px);
+  }
+
+  .grid-container {
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: repeat(4, auto);
+  }
+
+  .grid-layout {
+    display: block;
+  }
+
+  .layout1 {
+    grid-row: 2;
+    grid-column: 5/9;
+    background-image: url('/assets/img/grid-bg1.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: bottom;
+  }
+
+  .layout2 {
+    grid-row: 3/ span1;
+    grid-column: 7/-1;
+    background-image: url('/assets/img/grid-bg2.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: 0 190px;
+  }
+
+  .layout3 {
+    grid-row: 4/span1;
+    grid-column: 1/3;
+    background-image: url('/assets/img/grid-bg3.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: 0 590px;
+  }
+
+  .layout4 {
+    grid-row: 4/span1;
+    grid-column: 4/7;
+    background-image: url('/assets/img/grid-bg4.png');
+    background-repeat: no-repeat;
+    background-size: 16vh;
+    background-position: right 590px;
+  }
+
+  .card1 {
+    grid-column: 5/-1;
+  }
+
+  .image1 {
+    height: 500px;
+    background-size: cover;
+  }
+
+  .card2 {
+    grid-column: 1/5;
+    margin-top: 5em;
+  }
+
+  .image2 {
+    height: 300px;
+    background-size: cover;
+  }
+
+  .card3 {
+    grid-row: 2;
+    grid-column: 9/-1;
+    margin-top: 4em;
+  }
+
+  .image3 {
+    background-size: cover;
+    height: 400px;
+  }
+
+  .card4 {
+    grid-column: 1/7;
+    transform: translateY(-85px);
+  }
+
+  .image4 {
+    height: 50vh;
+    background-size: cover;
+  }
+
+  .card5 {
+    grid-column: 7/-1;
+    transform: translateY(-85px);
+  }
+
+  .image5 {
+    height: 50vh;
+    background-size: cover;
+  }
+
+  .card6 {
+    display: none;
+  }
+
+  .card-btn {
+    width: 12em;
+  }
```

Comment:
- [ ] Kindly update the button colors to match Figma design.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/187005852-d65cb4db-6062-4209-b5eb-fb19bcf7d5a2.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/187005863-d8a4033c-de85-4165-a1f0-5676afb4f996.png)